tests/Unit/IO/H5/Test_H5.cpp

```diff
@@ -646,39 +646,55 @@ SPECTRE_TEST_CASE("Unit.IO.H5.contains_attribute_false", "[Unit][IO][H5]") {
   CHECK_H5(H5Fclose(file_id), "Failed to close file: '" << h5_file_name << "'");
 }
 
-// [[OutputRegex, Failed HDF5 operation: Failed to open dataset 'no_dataset']]
-SPECTRE_TEST_CASE("Unit.IO.H5.read_data_error", "[Unit][IO][H5]") {
-  ERROR_TEST();
-  const std::string file_name("Unit.IO.H5.read_data_error.h5");
-  const hid_t file_id = H5Fcreate(file_name.c_str(), h5::h5f_acc_trunc(),
-                                  h5::h5p_default(), h5::h5p_default());
-  CHECK_H5(file_id, "Failed to open file: " << file_name);
-  static_cast<void>(h5::read_data<1, DataVector>(file_id, "no_dataset"));
-  CHECK_H5(H5Fclose(file_id), "Failed to close file: '" << file_name << "'");
-}
-
-// [[OutputRegex, Cannot create group 'group' in path: /group because the access
-// is ReadOnly]]
-SPECTRE_TEST_CASE("Unit.IO.H5.OpenGroup_read_access", "[Unit][IO][H5]") {
-  ERROR_TEST();
-  const std::string file_name("Unit.IO.H5.OpenGroup_read_access.h5");
-
-  const hid_t file_id = H5Fcreate(file_name.c_str(), h5::h5f_acc_trunc(),
-                                  h5::h5p_default(), h5::h5p_default());
-  CHECK_H5(file_id, "Failed to open file: " << file_name);
-  { h5::detail::OpenGroup group(file_id, "/group", h5::AccessType::ReadOnly); }
-  CHECK_H5(H5Fclose(file_id), "Failed to close file: '" << file_name << "'");
-}
-
-// [[OutputRegex, Failed to open the group 'group' because the file_id passed in
-// is invalid, or because the group_id inside the OpenGroup constructor got
-// corrupted. It is most likely that the file_id is invalid.]]
-SPECTRE_TEST_CASE("Unit.IO.H5.OpenGroup_bad_group_id", "[Unit][IO][H5]") {
-  ERROR_TEST();
-  h5::detail::OpenGroup group(-1, "/group", h5::AccessType::ReadWrite);
+namespace {
+void test_some_more_errors() {
+  // Failed HDF5 operation: Failed to open dataset 'no_dataset']]
```

Comment:
Remove leftover comment fragment.  This happens several more times in this file.